### ISSUE TITLE
test failure with perl-5.38.0-RC0 (t/20_unknown.t)

### ISSUE THREAD

**Issue by akhuettel** [rurban/Cpanel-JSON-XS#222]
In a linux environment testing Perl 5.38.0-RC0 I got the following test failure. More info on request...

```
#   Failed test 'pp sv_yes'
#   at t/20_unknown.t line 53.

#   Failed test 'pp sv_no'
#   at t/20_unknown.t line 54.
#          got: '{"false":false}'
#     expected: '{"false":""}'

#   Failed test 'pp sv_no'
#   at t/20_unknown.t line 55.
#          got: '{"false":false}'
#     expected: '{"false":""}'
# Looks like you failed 3 tests of 32.
```

```
Test Summary Report
-------------------
t/20_unknown.t           (Wstat: 768 (exited 3) Tests: 32 Failed: 3)
  Failed tests:  15-17
  Non-zero exit status: 3
Files=60, Tests=2847,  1 wallclock secs ( 0.30 usr  0.07 sys +  4.01 cusr  0.93 csys =  5.31 CPU)
Result: FAIL
Failed 1/60 test programs. 3/2847 subtests failed.
make: *** [Makefile:1217: test_dynamic] Error 255
 * ERROR: dev-perl/Cpanel-JSON-XS-4.290.0::gentoo failed (test phase):
```

**Comment by akhuettel** [comment]
Note, this was 4.29 and from the ChangeLog the issue may already have been fixed. Packaging 4.36 now, results upcoming.

**Comment by akhuettel** [comment]
Sorry for the spam. Fixed in 4.36.
